viahtml/views/blocklist.py

```diff
@@ -46,12 +50,17 @@ def __call__(self, path, environ, start_response):
         if not url:
             return None
 
-        reason = self.blocklist.is_blocked(url)
-        if not reason:
+        try:
+            hits = self.checkmate.check_url(url)
+        except CheckmateException as err:
+            LOG.warning("Failed to check URL against Checkmate: %s", err)
+            return None
```

Comment:
Whine and continue if Checkmate is down

Reply:
Does checkmate have a tight timeout on these requests, as well?